src/lib/RequestParser.ts

```diff
@@ -191,18 +242,29 @@ export class RequestParser {
                     flags: signTransactionRequest.flags,
                     validityStartHeight: signTransactionRequest.validityStartHeight,
                 } as SignTransactionRequest;
-            case RequestType.CASHLINK:
-                const cashlinkRequest = request as ParsedCashlinkRequest;
+            case RequestType.CREATE_CASHLINK:
+                const createCashlinkRequest = request as ParsedCreateCashlinkRequest;
+                // Note that there is no need to export autoTruncateMessage as the message already got truncated
                 return {
-                    appName: cashlinkRequest.appName,
-                    cashlinkAddress: cashlinkRequest.cashlinkAddress
-                            ? cashlinkRequest.cashlinkAddress.toUserFriendlyAddress()
-                            : undefined,
-                    senderAddress: cashlinkRequest.senderAddress
-                            ? cashlinkRequest.senderAddress.toUserFriendlyAddress()
+                    appName: createCashlinkRequest.appName,
+                    senderAddress: createCashlinkRequest.senderAddress
+                            ? createCashlinkRequest.senderAddress.toUserFriendlyAddress()
```

Comment:
This and similar occurences can be shortened to
```ts
senderAddress: createCashlinkRequest.senderAddress?.toUserFriendlyAddress(),
```
with [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) in Typescript 3.7.